File: Basics_with_Python/PythonBasicCourse/2 Conditional Statements/6_Conditional_Statements_Excercises - Exercise/more_exercises/08. Fuel Tank - Part 2.py
fuel_type = input()
fuel = float(input())
card = input()
price = 0
if fuel_type == "Diesel":
    if card == "Yes":
        price = fuel * 2.33 - fuel * 0.12
    elif card == "No":
        price = fuel * 2.33

elif fuel_type == "Gasoline":
    if card == "Yes":
        price = fuel * 2.22 - fuel * 0.18
    elif card == "No":
        price = fuel * 2.22

elif fuel_type == "Gas":
    if card == "Yes":
        price = fuel * 0.93 - fuel * 0.08
    elif card == "No":
        price = fuel * 0.93
if fuel >= 20 and fuel <= 25:
    price = price * 0.92
elif fuel > 25:
    price = price * 0.9
print(f"{price:.2f} lv.")

File: Basics_with_JavaScript/56_Conditional_Statements/More_Exersises/08._Fuel_Tank___Part_2.js
function vladi(input) {
  let gasolinePrise = 2.22;
  let dieselPrice = 2.33;
  let gasPrice = 0.93;
  let fuelType = String(input[0]);
  let liters = Number(input[1]);
  let command = String(input[2]);
  let price = 0;

  if (command === "Yes") {
    gasolinePrise -= 0.18;
    dieselPrice -= 0.12;
    gasPrice -= 0.08;
  }

  if (fuelType === "Gas") {
    price = gasPrice;
  } else if (fuelType === "Gasoline") {
    price = gasolinePrise;
  } else if (fuelType === "Diesel") {
    price = dieselPrice;
  }
  let bill = price * liters;
  if (liters >= 20 && liters <= 25) {
    bill *= 0.92;
  } else if (liters > 25) {
    bill *= 0.9;
  }
  bill = bill.toFixed(2);
  console.log(`${bill} lv.`);
}

vladi(["Gas", "30", "Yes"]);
vladi(["Gasoline", "25", "No"]);
vladi(["Diesel", "19", "No"]);
